ddtrace/internal/runtime/collector.py

```diff
@@ -1,4 +1,8 @@
 import importlib
+from typing import Collection
```

Comment:
`Collection` was added in Python 3.6 and deprecated in 3.9: https://docs.python.org/3/library/typing.html#typing.Collection